sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_receiver.py

```diff
@@ -350,11 +358,14 @@ def _create_handler(self, auth):
             timeout=self._max_wait_time * 1000 if self._max_wait_time else 0,
             prefetch=self._prefetch_count,
             # If prefetch is 1, then keep_alive coroutine serves as keep receiving for releasing messages
-            keep_alive_interval=self._config.keep_alive if self._prefetch_count != 1 else 5,
+            keep_alive_interval=self._config.keep_alive
+            if self._prefetch_count != 1
+            else 5,
             shutdown_after_timeout=False,
+            link_properties={CONSUMER_IDENTIFIER: self._name},
```

Comment:
Do we need to escape the _name somehow? Can a caller provide a name with a record separator etc. that would mess up things badly?

Reply:
I checked with Jessie and there aren't any restrictions or reserved characters from the service side as to what can be in that id. I tried a few id's and SB accepted without complaining and dotnet has exposed this for a few years now without escaping as well.

Reply:
..and should we access the attribute directly or through the `identifier` property?